ui/src/components/BaseballCard.jsx

```diff
@@ -26,17 +27,26 @@ const BaseballCard = ({ imageOnly, playerName, handleClick }) => {
       >
         <img
           className="h-full w-full rounded-md"
-          src={images[playerName]}
+          src={`https://gateway.pinata.cloud/ipfs/${playerName.image}`}
```

Comment:
Check whether an image is an ipfs hash or an https url. Only use gateway if an ipfs hash